exonum-java-binding/core/src/main/java/com/exonum/binding/core/transaction/TransactionMethod.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.core.transaction;
+
+import com.exonum.core.messages.Runtime.ErrorKind;
+import com.exonum.core.messages.Runtime.ExecutionError;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Indicates that a method is a transaction method. The annotated method should execute the
+ * transaction, possibly modifying the blockchain state. The method should:
+ * <ul>
+ *   <li>be public
+ *   <li>have exactly two parameters of types 'byte[]' and '{@link TransactionContext}' in this
```

Comment:
Parameter documentation (most importantly, what is `byte[]`, also what is `context` for)?
The `com.exonum.binding.core.service.TransactionConverter#toTransaction(int, byte[])` and `Transaction#execute` have it correspondingly.